### DISCUSSION TITLE
Gas Fees

### DISCUSSION THREAD

**Discussion by JacobEverly** [ava-labs/subnet-evm#966] [General]
In the Genesis config for EVM, we have in our docs that min/maxBlockGasCost : Sets the minimum/maxmium amount of gas to charge for the production of a block.

For the gas that is paid for the block production, I assume that the amount of gas contributed to this total amount varies by each transaction? Is that correct.

**Comment by aaronbuchwald** [comment]
Yes, that's correct.

The contribution towards the block gas cost is the total amount spent above what was required for inclusion: (effectiveGasPrice - baseFee) * gasUsed

https://github.com/ava-labs/subnet-evm/blob/master/consensus/dummy/consensus.go#L295-L313
